README.md

```diff
@@ -37,15 +37,13 @@ In both modes, system-level metrics (e.g. tracepoints), are always grouped by th
 
 # Build Requirements
 
- * Linux<sup>1</sup>
+ * Linux (>= 4.3)
  * [OTF2](http://www.vi-hps.org/projects/score-p/index.html) (>= 3.0)
  * libbfd
  * libiberty
  * CMake (>= 3.11)
  * A C++ Compiler with C++17 support and the std::filesystem library (GCC > 7, Clang > 5)
 
-<sup>1</sup>: Use Linux >= 4.1 for best results. Older versions, even the ancient 2.6.32, will work, but with degraded time synchronization.
```

Comment:
We could mention that lo2s versions before $NEXT_VERSION can be used for older kernels.

As we do breaking changes right now, $NEXT_VERSION will be 2.0, right?

Reply:
Yeah, we should release 2.0 following this.